matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/aggregation/poll/DefaultPollAggregationProcessor.kt

```diff
@@ -156,7 +156,7 @@ class DefaultPollAggregationProcessor @Inject constructor() : PollAggregationPro
     override fun handlePollEndEvent(session: Session, powerLevelsHelper: PowerLevelsHelper, realm: Realm, event: Event): Boolean {
         val content = event.getClearContent()?.toModel<MessageEndPollContent>() ?: return false
         val roomId = event.roomId ?: return false
-        val pollEventId = content.relatesTo?.eventId ?: return false
+        val pollEventId = (event.getRelationContent() ?: content.relatesTo)?.eventId ?: return false
```

Comment:
does `content.relatesTo` has any sense here? Won't it be returned from `event.getRelationContent()` if exists?

Reply:
Yes you are right. If `getRelationContent` returns null there is little chance `content.relatesTo` returns something. I will update.

Reply:
Updated here: [71df1e6](https://github.com/vector-im/element-android/pull/7757/commits/71df1e61d43782f7a181f267ab75e1f173dbbc0a)